lib/asciidoctor/converter/manpage.rb

```diff
@@ -201,6 +201,9 @@ def convert_dlist node
         result << %(.sp
 #{counter}. #{manify terms.map {|dt| dt.text }.join ' '}
 .RS 4)
+      when 'horizontal'
+        result << %(.TP
+#{manify terms.map {|dt| dt.text }.join(', '), whitespace: :normalize})
```

Comment:
I tried this and it doesn't seem to have any effect. Can you clarify how the TP macro works?